sdk/lib/stellar/horizon_error.rb

```diff
@@ -0,0 +1,53 @@
+module Stellar
+  class HorizonError < StandardError
+    attr_reader :original_response, :extras
+
+    def self.make(hash)
```

Comment:
Since we then assigned the `hash` parameter to the instance variable `original_response`, I believe it's better to name it the same here. Also it would make it more accurate. The same goes for the `initialize` method